### PR TITLE
sliding DPI shows suitable keys

### PR DESCRIPTION
**Pull request by pfps**
@barolo @sk8geek @mschilli87 @Vtec234 Please test this PR if possible.

Test with a mouse that has the Sensitivity (DPI) setting, such as the MX Master 3 or the MX Vertical.

These mice should have a DPI Sliding Adjustment.  If the mouse has a DPI Switch key, the choices should be Off and DPI Switch.  If the mouse does not have this key, the choices should be Off and the keys that can send raw xy mouse movements.   For the MX Master 3 they are middle, back, forward, gesture, and smart shift.  

Change the setting to one of the keys.  Click the button, move the mouse an inch or two, and release the button.   The mouse pointer should not move.  As the mouse moves notifications showing a DPI value should appear.  When the mouse is released the Sensitivity (DPI) value should be changed to this value and the sensitivity of the mouse should change.

Note:  If you go back to a previous version of Solaar you may encounter errors on startup.  Go in to ~/.config/solaar/config.json and remove the dpi-sliding value to fix this issue.

### CODE REVIEW

File: lib/logitech_receiver/settings_templates.py [hunk 1]
```diff
@@ -406,17 +400,6 @@ def __init__(self, device, dpiSetting):
                 we go into the `moved` state.
                 When the button is released in this state
                 the DPI is set according to the total displacement.
-
```
Comment (Vtec234) [inline lib/logitech_receiver/settings_templates.py:409]: Why remove this beauty of a diagram

Comment (pfps) [reply lib/logitech_receiver/settings_templates.py:409]: I'm in favour of brevity and the FSM is described in text.

**Comment by Vtec234** [comment]
Seems to work on MX Vertical. The options are Off and DPI Switch, the sliding works with DPI Switch selected while with Off nothing happens (as it should, I guess).

**Comment by sk8geek** [comment]
(I don't have an "MX" mouse.)

**Pull request merged**